src/styles/components/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  width: 100px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-around;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    box-sizing: border-box;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: left;
```

Comment:
The `background-position` property should be set to `center` instead of `left` to ensure that the star images are centered within their container elements, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).